library/Fission/Prelude.hs

```diff
@@ -15,19 +17,68 @@ module Fission.Prelude
   , putText
   , putTextLn
   , textShow
+  , logInfo
+  , logDebug
+  , logWarn
+  , logError
+  , logOther
   ) where
 
-import Control.Lens                ((%~), (.~), (?~), (^?))
+import Control.Lens         ((%~), (.~), (?~), (^?))
+import Control.Monad.Logger (MonadLogger (..), LogSource, ToLogStr (..), LogLevel (..), logWithoutLoc)
+import Control.Monad.Time
+
+import Network.IPFS.Internal.Orphanage.Utf8Builder ()
+
 import Data.Aeson
 import Data.Has
 import Data.Maybe
-import Fission.Internal.Constraint
-import Fission.Internal.UTF8       (putText, putTextLn, textShow)
+
 import Flow
-import RIO                         hiding (Handler, id, timeout, ($), (&))
-import RIO.List                    (headMaybe, intercalate)
+
+import RIO.List (headMaybe, intercalate)
 import RIO.Process
 import RIO.Time
 
+import RIO.Orphans ()
+
+import RIO hiding ( Handler
+                  , LogLevel (..)
+                  , LogSource
+                  , logDebug
+                  , logDebugS
+                  , logError
+                  , logErrorS
+                  , logInfo
+                  , logInfoS
+                  , logOther
+                  , logOtherS
+                  , logWarn
+                  , logWarnS
+                  , id
+                  , timeout
+                  , ($)
+                  , (&)
+                  , (^.)
+                  )
+
+import Fission.Internal.UTF8       (putText, putTextLn, textShow)
+import Fission.Internal.MonadDB
+
 identity :: a -> a
 identity a = a
+
+logInfo :: (ToLogStr msg, MonadLogger m) => msg -> m ()
+logInfo = logWithoutLoc "" LevelInfo
+
+logDebug :: (ToLogStr msg, MonadLogger m) => msg -> m ()
+logDebug = logWithoutLoc "" LevelDebug
+
+logWarn :: (ToLogStr msg, MonadLogger m) => msg -> m ()
+logWarn = logWithoutLoc "" LevelWarn
+
+logError :: (ToLogStr msg, MonadLogger m) => msg -> m ()
+logError = logWithoutLoc "" LevelError
+
+logOther :: (ToLogStr msg, MonadLogger m) => LogLevel -> msg -> m ()
+logOther = logWithoutLoc ""
```

Comment:
Making the `MonadLogger` class play nice with `RIO`'s concrete logger. Renaming so we don't have to make any changes to the code 👍

However, because of how `MonadLogger` works, you will sometimes have to tell these functions what kind of string you're giving it, because it's no longer going to force you to use `Utf8Builder` — but that's also really great in a lot of places! Way less `UTF8.textShow` and whatnot. (This is because it has that `ToLogStr` constraint rather than a concrete type).